### ISSUE TITLE
[Spotlight] Digital Collection Home Page

### ISSUE THREAD

**Issue by devangm** [TAMULib/spotlight#97]
**URL:** https://spotlight.library.tamu.edu/spotlight/alice-in-wonderland
**WCAG 2.1 AA compliant? :** No
**Violation:** 2.4.5 Multiple Ways (AA)
**Issues:**  No way to navigate back to the main spotlight home page
**Notes:**
